src/transactions/TransactionFrame.cpp

```diff
@@ -818,7 +818,6 @@ TransactionFrame::refundSorobanFee(AbstractLedgerTxn& ltxOuter,
     auto feeSourceAccount = loadAccount(ltx, header, feeSource);
     if (!feeSourceAccount)
     {
-        // Account was merged (shouldn't be possible)
```

Comment:
nit: I think 'Account was merged' is still a useful comment; only the 'shouldn't be possible' part is incorrect.

Reply:
Yeah good point. Added it back.